topology/probes/k8s/graph.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 IBM Corp.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package k8s
+
+import (
+	"github.com/skydive-project/skydive/topology/graph"
+)
+
+func newMetadata(typ, name string, extra interface{}) graph.Metadata {
+	return graph.Metadata{
+		"Type":    typ,
+		"Probe":   "k8s",
+		"Manager": "k8s",
+		"Name":    name,
+		"K8s":     extra,
+	}
+}
+
+func addMetadata(g *graph.Graph, n *graph.Node, extra interface{}) {
+	g.AddMetadata(n, "Manager", "k8s")
+	g.AddMetadata(n, "K8s", extra)
```

Comment:
Same here, not that we should put everything directly, but acceptable for now I guess

Reply:
yes, using transactions now